job-service/src/main/java/com/hpe/caf/services/job/api/JobsGetById.java

```diff
@@ -65,6 +65,7 @@ public static Job getJob(final String partitionId, String jobId) throws Exceptio
             job = databaseHelper.getJob(partitionId, jobId);
 
             LOG.debug("getJobById: Done.");
+            LOG.info("Job progression {}/{} status {}  progression {}", partitionId, jobId, job.getStatus(), job.getPercentageComplete());
```

Comment:
Did you mean to write progression twice?

Reply:
Yes, like "Job progression" as title for what follows, then "progression" as details. But if that's confusing, I can change

Reply:
Changing for:
            LOG.info("Job progress {}/{} status {}  progression {}", partitionId, jobId, job.getStatus(), job.getPercentageComplete());